Contributing.md

```diff
@@ -0,0 +1,176 @@
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **Include screenshots and/or animated GIFs if applicable** which show you following the described steps and clearly demonstrate the problem.
+* **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened and share more information using the guidelines below.
+
+Provide more context by answering these questions:
+
+* **Did the problem start happening recently** (e.g. after updating to the latest build of NovelRT) or was this always a problem?
+* If the problem started happening recently, **can you reproduce the problem in an older build of NovelRT?** What's the most recent build/commit SHA in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+* What area of NovelRT is the problem occurring in? (e.g. Is it Input related? Is it related to Rendering or Windowing? etc)
+
+Include details about your configuration and environment:
+
+* **Which build of NovelRT are you using?**
+* **What's the name and version of the OS you're using**?
+* **Was this pre-built? If not, what compiler was NovelRT built with?**
+* **Are you using NovelRT with multiple monitors?** If so, can you reproduce the problem when you use a single monitor?
+* **Which locale/region is your OS configured to use?**
+
+#### Invalid Reports
+
+While we understand that not every issue may get resolved, we have a zero-tolerance policy for issues that fall under the following categories:
+
+* Spam
+* Hateful/discriminatory language
+* Repeatedly reported non-issue
+
+Please note that while we will do our best to filter out these types of submissions, they may still pop up from time to time. If you notice that one of these types of issues are opened in _any_ of the NovelRT organisation's repositories, please e-mail [admin@novelrt.dev](mailto:admin@novelrt.dev) or message one of the maintainers in Discord. _Furthermore, any attempt to open an issue that falls under the above categories may result in disiplinary action by the NovelRT team up to and including a ban from interacting with the NovelRT repository (and other related repositories within the NovelRT organisation)_.
```

Comment:
We should probably add that even if it doesn't explicitly fall under this, we reserve the right to do it anyway if we feel you're in violation.

Reply:
`The NovelRT team also reserves the right to enforce disiplinary action for any reason that may violate our Code of Conduct at any time.`

Sounds good?

Reply:
Sounds good. :)